src/Tizen.Applications.Cion/Tizen.Applications/Cion/SecurityInfo.cs

```diff
@@ -0,0 +1,167 @@
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get CA path.");
+                    return "";
+                }
+                return caPath;
+            }
+
+            set
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecuritySetCaPath(_handle, value);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    throw CionErrorFactory.GetException(ret, "Failed to set CA path.");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets the cert path.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when the cert path is invalid.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public string CertPath
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecurityGetCertPath(_handle, out string certPath);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get cert path.");
+                    return "";
+                }
+                return certPath;
+            }
+
+            set
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecuritySetCertPath(_handle, value);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    throw CionErrorFactory.GetException(ret, "Failed to set cert path.");
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets the private key path.
+        /// </summary>
+        /// <exception cref="ArgumentException">Thrown when the private key path is invalid.</exception>
+        /// <since_tizen> 9 </since_tizen>
+        public string PrivateKeyPath
+        {
+            get
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecurityGetPrivateKeyPath(_handle, out string privateKeyPath);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    Log.Error(LogTag, "Failed to get private key path.");
+                    return "";
+                }
+                return privateKeyPath;
+            }
+
+            set
+            {
+                Interop.Cion.ErrorCode ret = Interop.CionSecurity.CionSecuritySetPrivateKeyPath(_handle, value);
+                if (ret != Interop.Cion.ErrorCode.None)
+                {
+                    throw CionErrorFactory.GetException(ret, "Failed to set private key path.");
+                }
+            }
+        }
+
+        #region IDisposable Support
+        private bool disposedValue = false;
+
+        /// <summary>
+        /// Releases any unmanaged resources used by this object. Can also dispose any other disposable objects.
```

Comment:
param is not described. Maybe something like this?
/// <param name="disposing">If true, disposes any disposable objects. If false, does not dispose disposable objects.</param>

Reply:
I added missing doxygen comment.